### ISSUE TITLE
Add compiler hints or recognition for tabs (\t)

### ISSUE THREAD

**Issue by way2muchnoise** [AfterLifeLochie/fontbox#9]
When rendering the line \t is ignored only \s get's used but that isn't usefull since boxText will trim everything. My advise would be add a tab size to a line or an x time space size. It would need work in the compiler too. So that words don't go out of the box and tabs at end of lines don't need to be handled.

**Comment by AfterLifeLochie** [comment]
So in other words, you'd like to be able to treat tabs as "fixed width non-breaking space" except for where the tab is on the end of the paginated line?

**Comment by way2muchnoise** [comment]
Yeah some thing like that

**Comment by AfterLifeLochie** [comment]
Right. I'll likely add a `tabSize` property on `PageProperties` to let you do this.
